packages/openapi-code-generator/src/core/openapi-types.ts

```diff
@@ -248,6 +248,9 @@ export interface Schema {
   externalDocs?: ExternalDocumentation | undefined
   deprecated?: boolean | undefined
   // xml?: XML | undefined
+
+  // TODO: not yet supported by type-builder or joi
+  "x-alpha-transform"?: string | undefined
```

Comment:
It would probably be more useful to have this be shaped like `{fn: string, type: string}` such that we can plumb the return type through the `type-builder` properly